### DISCUSSION TITLE
singleUse and Multiple use

### DISCUSSION THREAD

**Discussion by khalidzaibi** [ash-jc-allen/short-url#154] [Q&A]
the single use function is only for single time use, but multiple usage like ifi need to use a like 3 time or 5 times what would i do for that ?

**Comment by ash-jc-allen** [comment]
Hey! At the moment, there isn't any functionality built into the package to set limits for how many visits a short URL can have (apart from single-use).

So, I think if you wanted to implement this functionality, you'd be best defining a custom route and controller to handle the visits. In the controller, you could then decide whether to redirect or reject the visitor 🙂
